### ISSUE TITLE
FlexiSpot E7 Pin Map in Readme

### ISSUE THREAD

**Issue by bryanhunwardsen** [iMicknl/LoctekMotion_IoT#58]
I think the pin values in the pin mapping are inverted.
Using this:
![image](https://github.com/iMicknl/LoctekMotion_IoT/assets/3408302/cada14a4-c333-44f9-881b-47dd79233842)
White Orange is Pin 1
![IMG20230620113612](https://github.com/iMicknl/LoctekMotion_IoT/assets/3408302/6a2c6e15-1e39-4582-9e3f-2362072ba1d9)
This pic has the E7 Panel cable upside down from the above diagram to see the wire color, 
Yellow is on the left, but when flipped to match the orientation of the diagram it is pin 8.

**Comment by bryanhunwardsen** [comment]
Looks as pin numbers for all the maps are inverted.
This shows plug upside down from my pic, but indicates more so the pin id #'s need to be inverted in your readme.

![image](https://github.com/iMicknl/LoctekMotion_IoT/assets/3408302/2204d239-3fac-44a6-ac3c-b2f1e9104d40)

**Comment by bryanhunwardsen** [comment]
Sorry to many wires and cables upside down. Everything is correct spologies.

**Comment by CoderTobi** [comment]
I agree with you. The table in the readme file seems to be inverted or mirrored. 
I used a RJ45 Breakoutboard to connect the ESP to the table and used a multimeter to determine which pin is 5V.

The highlighted collum shows the correct pin config. The one right next to it is how its documented in the readme-file.
![image](https://github.com/iMicknl/LoctekMotion_IoT/assets/77673526/1cf35a8d-b8f5-4670-b7b1-67b676bbd8f6)

Since the table is apparently using 5V on the UART pins this is very unhealthy for the ESP as the TX and RX pins don't match in the documentation from the readme file.

**Comment by jamesmyatt** [comment]
I think I fixed this is #91 on the v2 branch.

**Comment by Jonas354** [comment]
Yes, same pin-out.
